### PR TITLE
[INFRA/RULES] Parameterised starting with rules in play

### PR DESCRIPTION
**Pull request by Neelesh99**
# Summary

Start with some rules in play, cos otherwise it's a lawless wasteland. Activated by parameter.

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: internal/server/server.go [hunk 1]
```diff
@@ -79,6 +80,11 @@ func createSOMASServer(
 			CommonPool:  gameConfig.InitialCommonPool,
 		},
 	}
+	if gameConfig.IIGOConfig.StartWithRulesInPlay {
+		for ruleName, _ := range rules.AvailableRules {
```
Comment (lhl2617) [inline internal/server/server.go]: can you see linter warnings? seems like most people can't because they are not fixed

Comment (Neelesh99) [reply internal/server/server.go]: XD, as in it doesn't show in IDE, I have to look at PR's to see them. Fixed in latest push

**Review by NGovani — APPROVED** [review]
LGTM, not a fan of  Available Rules and Rules In Play being package globals but if it works its all good

**Pull request merged**